docs/404.md

```diff
@@ -0,0 +1,19 @@
+---
+<!-- title: 404 -->
+id: 404
+layout: 404
+description: 404 page
+permalink: /404.html
+---
+
+<div class="four-o-four-wrapper">
+  <div class="four-o-four-title">
+    <h2>Oh no! I guess that page is a secret.</h2>
+  </div>
+  <div class="four-o-four-img">
+    <img class="404-img" src="/img/404_zipped.png" alt="404 image">
+  </div>
+  <div class="four-o-four-btn">
+    <span class="arch-callout ">Back to Homepage</span>
```

Comment:
I expected this to be a button that I could click on to go back to the homepage, but it isn't - can we make it one?